.gitignore

```diff
@@ -1,2 +1,3 @@
 .direnv
-result
\ No newline at end of file
+result
+.pre-commit-config.yaml
```

Comment:
Also, I think this has to be unchecked for the configuration to work, right?

Reply:
That's not necessarily the case. The config file is generated by [pre-commit.installationScript](https://flake.parts/options/pre-commit-hooks-nix.html#opt-perSystem.pre-commit.installationScript), which I have included in `perSystem.shellhook`. So, every time `nix develop` is run, it sets up pre-commit locally with hooks that have the appropriate paths to /nix/store in the config file.

<details>
<summary>.pre-commit-config.yaml</summary>

    # DO NOT MODIFY
    # This file was generated by pre-commit-hooks.nix
    {
      "default_stages": [
        "commit"
      ],
      "repos": [
        {
          "hooks": [
            {
              "entry": "/nix/store/7za878069xw6c2qh9yhc6cdnbxbdn2yr-nixpkgs-fmt-1.3.0/bin/nixpkgs-fmt",
              "exclude": "^$",
              "files": "\\.nix$",
              "id": "nixpkgs-fmt",
              "language": "system",
              "name": "nixpkgs-fmt",
              "pass_filenames": true,
              "stages": [
                "commit"
              ],
              "types": [
                "file"
              ],
              "types_or": []
            },
            {
              "entry": "/nix/store/g74w0iw2qp43yq55bv1ly19zgzli2i89-shellcheck-0.9.0-bin/bin/shellcheck",
              "exclude": "^$",
              "files": "",
              "id": "shellcheck",
              "language": "system",
              "name": "shellcheck",
              "pass_filenames": true,
              "stages": [
                "commit"
              ],
              "types": [
                "shell"
              ],
              "types_or": [
                "sh",
                "ash",
                "bash",
                "bats",
                "dash",
                "ksh"
              ]
            }
          ],
          "repo": "local"
        }
      ]
    }

</details>

Reply:
Ok, then I have misunderstood. This is fine as-is then.